tutorials/ui/gui_skinning.rst

```diff
@@ -1,183 +1,240 @@
+   :align: center
 
-In the theme editor, next to the default preview window, is where you make changes
-to your theme. Clicking the plus button opens the **Add item Type** menu.
+   A "Gear Up!" screen in Tank Kings, courtesy of Winterpixel Games
 
-.. image:: img/add_item_type.png
+Beyond achieving a unique look for your game this system also enables developers
+to provide customization options to the end users, including accessibility settings.
+UI themes are applied in a cascading manner (i.e. they propagate from parent
+controls to their children), which means that font settings or adjustments for
+colorblind users can be applied in a single place and affect the entire UI tree.
+Of course this system can also be used for gameplay purposes, like hero character
+styling or team colors and other visual settings.
 
-From here select the control node you want your theme to modify and click **Ok**. You
-should now see theme items for that node in the theme editor. Theme items are what defines
-the look of a theme, each kind of item in a theme can be:
-
--  **Color**: A single color, with or without transparency. Colors are
-   usually applied to fonts and icons.
--  **Constant**: A single numerical constant. Generally used
-   to define spacing between components or alignment.
--  **Font**: Every control that uses text can be assigned the fonts
-   used to draw strings.
--  **Icon**: A single image. Textures are not often used, but when
-   they are, they represent handles to pick or icons in a complex control
-   (such as a file dialog).
--  **StyleBox**: Stylebox is a resource that defines how to draw a
-   panel in varying sizes (more information on them later).
-
-Every item is associated with:
+Basics of themes
+----------------
 
--  A name (the name of the item)
--  A Control (the name of the control)
+The skinning system is driven by the :ref:`Theme <class_Theme>` resource. Every
+Godot project has an inherent default theme that contains the settings used by
+the built-in control nodes. This is what gives the controls their distinct look
+out of the box. A theme only describes the configuration, however, and it is still
+the job of each individual control to use that configuration in the way it requires
+to display itself. This is important to remember when implementing
+:ref:`your own custom controls <doc_custom_gui_controls>`.
 
-To customize a theme item click on the plus sign next to it. Your theme
-will now override the default theme for that item. To modify it click on **Empty**,
-then select the new theme item you want to create. Click on it again and you can
-now modify it in the inspector.
+.. note::
+   Even the Godot editor itself relies on the default theme. But it doesn't look the
+   same as a Godot project, because it applies its own heavily customized theme on top
+   of the default one. In principle this works exactly like it would in your game,
+   which is explained in a :ref:`section below <doc_gui_theme_in_project>`.
```

Comment:
```suggestion
   of the default one. In principle, this works exactly like it would in your game
   as explained :ref:`below <doc_gui_theme_in_project>`.
```